src/DOM/DOMRegistry.js

```diff
@@ -7,253 +7,254 @@ const xlink = 'http://www.w3.org/1999/xlink';
 const xml = 'http://www.w3.org/XML/1998/namespace';
 
 const DOMAttributeNamespaces = {
-    'xlink:actuate': xlink,
-    'xlink:arcrole': xlink,
-    'xlink:href': xlink,
-    'xlink:role': xlink,
-    'xlink:show': xlink,
-    'xlink:title': xlink,
-    'xlink:type': xlink,
-    'xml:base': xml,
-    'xml:lang': xml,
-    'xml:space': xml
+	'xlink:actuate': xlink,
+	'xlink:arcrole': xlink,
+	'xlink:href': xlink,
+	'xlink:role': xlink,
+	'xlink:show': xlink,
+	'xlink:title': xlink,
+	'xlink:type': xlink,
+	'xml:base': xml,
+	'xml:lang': xml,
+	'xml:space': xml
 };
 
 const DOMAttributeNames = {
-    acceptCharset: 'accept-charset',
-    className: 'class',
-    htmlFor: 'for',
-    httpEquiv: 'http-equiv',
-    xlinkActuate: 'xlink:actuate',
-    xlinkArcrole: 'xlink:arcrole',
-    xlinkHref: 'xlink:href',
-    xlinkRole: 'xlink:role',
-    xlinkShow: 'xlink:show',
-    xlinkTitle: 'xlink:title',
-    xlinkType: 'xlink:type',
-    xmlBase: 'xml:base',
-    xmlLang: 'xml:lang',
-    xmlSpace: 'xml:space',
-    viewBox: 'viewBox' // SVG - Edge case. The letter 'b' need to be uppercase
+	acceptCharset: 'accept-charset',
```

Comment:
@hbstone I say concretely. Why 1 space here, for example? It's difficult to read...